docs/release-notes/release-notes-0.19.0.md

```diff
@@ -224,6 +224,23 @@ The underlying functionality between those two options remain the same.
 
 
 ## Breaking Changes
+
+### ⚠️ **Warning:** The following RPCs will be removed in release version **0.21**:
+
+| Deprecated RPC Method | REST Equivalent | HTTP Method | Path | Replaced By |
+|----------------------|----------------|-------------|------------------------------|------------------|
+| [`SendToRoute`](https://lightning.engineering/api-docs/api/lnd/lightning/send-to-route/index.html) | ❌ (No direct REST equivalent) | — | — | [`SendToRouteV2`](https://lightning.engineering/api-docs/api/lnd/router/send-to-route-v2/) |
```

Comment:
I think we'll also want to remove the `routerrpc.SendToRoute`, not just the `lnrpc.SendToRoute`. So herhaps we should mention both here and explicitly prefix them to distinguish. Same with `routerrpc.SendPayment` vs. `lnrpc.SendPayment`.